src/ios/FirebaseCrashPlugin.m

```diff
@@ -0,0 +1,41 @@
+#import "FirebaseCrashPlugin.h"
+#import <Fabric/Fabric.h>
+#import <Crashlytics/Crashlytics.h>
+
+@implementation FirebaseCrashPlugin
+
+- (void)pluginInitialize {
+    NSLog(@"Starting Firebase Crash plugin");
+}
+
+- (void)log:(CDVInvokedUrlCommand *)command {
+    [self.commandDelegate runInBackground:^{
+        NSString* errorMessage = [command.arguments objectAtIndex:0];
+        CLSNSLog(@"%@", errorMessage);
+        CDVPluginResult *pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK];
+        [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+    }];
+}
+
+- (void)sendNonFatalCrash:(CDVInvokedUrlCommand*)command {
+    NSString *description = NSLocalizedString([command argumentAtIndex:0 withDefault:@"No Message Provided"], nil);
```

Comment:
4 spaces for tabulation

Reply:
Think it is already 4 spaces or am I wrong?